pkg/admissioncontroller/webhooks/admission/auditpolicy/admission.go

```diff
@@ -75,8 +73,6 @@ func New(logger logr.Logger) *handler {
 func init() {
 	auditPolicyScheme := runtime.NewScheme()
 	schemeBuilder := runtime.NewSchemeBuilder(
-		auditv1alpha1.AddToScheme,
-		auditv1beta1.AddToScheme,
```

Comment:
This means that we don't accept audit configurations in `v1alpha1` or `v1beta1` anymore. Probably we should add this explicitly as a release note.